po/nb.po

```diff
@@ -125,13 +124,12 @@ msgid "Open datafile read-only"
 msgstr "Åpne datafil skrivebeskyttet"
 
 #: 2.6-release-tour.phtml:87
-#, fuzzy
 msgid ""
 "When opening a locked \"book\" one now has the option of opening it read-"
 "only."
 msgstr ""
 "Ved åpning av låst \"bokføring\" har man nå muligheten til å åpne den "
```

Comment:
@comradekingu Sorry for the intrusion. When I went through multiple-errors to search for source file weaknesses, I thought I append the full stop. But now I doubt "bokføring" is the right term here.

Reply:
Depends what "book" is supposed to mean. Translates to ~"bookkeeping"

Reply:
In this case "a file containing one book" is meant.